### DISCUSSION TITLE
md5sum - CDE 11556150

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#56] [Metadata Field Ratification]
caDSR Link: [11556150](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=11556150%20and%20ver_nr=1)

Definition: A 32-character hexadecimal number that is computed on a file.

Discussion: Not as secure as SHA-256 or SHA-512.  md5sum is required for dbGaP and SRA submissions.  md5sum is sufficient for ensuring a file was downloaded without loss.

**Comment by calkinsh** [comment]
CHOP approves this

**Comment by Ssandor13** [comment]
St. Jude's data pipeline currently does not save the md5sums, so we would not be able to provide them at this moment. But it’s something that we now have a use case for, so, we will have to work internally to generate and provide them for all of our samples in the future.

**Comment by GeoffLyle** [comment]
CDE accepted at January 25th meeting.

While all files may not immediately have a md5sum associated with them, the goal is to generate this data for all files shared via the Federation API.

**Comment by claymcleod** [comment]
I noticed this CDE's status is `DRAFT NEW`—do we need to ask them to officially adopt it?
